README.md

```diff
@@ -45,23 +45,45 @@ Supported Cars
   - Due to limitations in steering firmware, steering is disabled below 12 mph
   - Uses stock Honda Sensing for longitudinal control
 
-- Honda CR-V Touring 2015-2016
+- Honda CR-V Touring 2015-2016 with Honda Sensing
```

Comment:
It was intentional to leave off "with Honda Sensing". This is redundant because Honda Sensing is not an option, it is standard on touring trim, and touring only for these years

Reply:
The whole file is redundant.  The supported vehicles don’t belong on the readme.  They belong on the wiki or a linked `md` so we can dig deeper.

The code base has vehicles hardcoded.  If we’re writing if car.PILOT we are doing it wrong in the end.

It must be feature based or at the very least year make model based.  I get that it was mostly for rapid prototyping purposes but it’s all redundant in the end game.

To truly inspire, we will have to move away from the majority of how things are written.  But baby steps I suppose.  :D

Reply:
README went through big changes for 0.5 release. Better to revert the changes to this file.

Reply:
Fair enough, will do!